web-registry/src/pages/CreditClassDetails/CreditClassDetails.tsx

```diff
@@ -161,6 +161,8 @@ function CreditClassDetails({
             ? () => setIsBuyFlowStarted(true)
             : () => setDisplayErrorBanner(true)
         }
+        onChainCreditClassId={onChainClass?.id}
+        creditClassName={metadata?.['schema:name']}
```

Comment:
this how we pass in the values to the sticky nav buy button action for the credit class page.